cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -181,9 +181,9 @@ The workflow itself uses the same names in its configuration::
 
 Concurrent Workflows
 --------------------
-By default, a workflow runs sequentially. Multiple instances of a workflow can be run
-concurrently. To enable concurrent runs for a workflow, set its runtime argument
-``concurrent.runs.enabled`` to ``true``.
+By default, a workflow runs concurrently, allowing multiple instances of a workflow to be run
+simultaneously. To disable concurrent runs for a workflow, set its runtime argument
+``concurrent.runs.enabled`` to ``false``.
```

Comment:
This can be updated to `By default, a workflow runs concurrently, allowing multiple instances of a workflow to be run
 simultaneously. However, for the scheduled workflows, the number of concurrent runs can be controlled by <link_to_run_constraint>`